llvm/lib/Target/AMDGPU/LDSDIRInstructions.td

```diff
@@ -1,4 +1,4 @@
-//===-- LDSDIRInstructions.td - LDS Direct Instruction Definitions --------===//
+//===-- LDSDIRInstructions.td - LDS/DS Direct Instruction Definitions -----===//
```

Comment:
LDS/VDS

While I think it makes sense to keep these encodings in the same file as they essentially do the same thing, I suspect we should rename to file itself to DSDIRInstructions.td.
Assuming like the definitions before we consider DSDIR to be a superclass of LDSDIR and VDSDIR.